src/utils/gatewayHelpers.ts

```diff
@@ -25,3 +25,10 @@ export async function getUniqueName(client: any, name: string) {
     return name.toLowerCase();
   }
 }
+
+export async function getUniqueDomainName(client, solution_type, name) {
+  let twin_id = await client.twins.get_my_twin_id();
+  let domainName = `${solution_type}${twin_id}${name}`;
+  domainName = await getUniqueName(client, domainName);
```

Comment:
I don't think we need to use `getUniqueName` anymore as deployment names cannot be the same anyway

Also, its implementation actually covers only 10 possibilities, can be removed completely if not used anywhere.

Reply:
the `geUnigueName` is a recursive function. it will keep checking if the name exists will append one digit to it.